### DISCUSSION TITLE
change screenshot shortcut program

### DISCUSSION THREAD

**Discussion by brupec** [BartoszCichecki/LenovoLegionToolkit#791] [General]
Hello,

I was wondering if I can change the fn+prtscrn function, to launch any program I want (in my case sharex)
I tried to replace the exe of the default windows screenshot tools, but didn't manage to get the rights in windowsapp folder.

when pressing print screen without fn, sharex launch, but I use FnLock, so when I press print screen, the windows screenshot tools open.

I was wondering if anything can be done to change this behaviour ?

Thanks and regards

**Comment by BartoszCichecki** [comment]
Which laptop do you have?

**Reply by brupec** [reply]
Hi, Legion 5 15ACH6H.
BTW thanks for your great soft 😊

**Reply by BartoszCichecki** [reply]
At the moment it isn't possible but I'll take a look into what can be done. This key is handled differently on different models, so I can't promise anything tho.

**Reply by brupec** [reply]
Hi! I now have Legion Slim 5 16IRH8 (2023), by any chance can it be handled on this model ?

Thanks

**Comment by brupec** [comment]
Thanks a lot! I tried to detect the key code in some soft but I couldn't
get anything... Maybe you'll have more chance!

Le lun. 19 juin 2023, 15:50, Bartosz Cichecki ***@***.***> a
écrit :

**Comment by BartoszCichecki** [comment]
It looks like to implement it properly, LLT would need to intercept Win+Shift+S on most models, which is not really something that is viable option.
